### DISCUSSION TITLE
Skipping until all videos are processed for the artist

### DISCUSSION THREAD

**Discussion by astro7467** [RandomNinjaAtk/docker-lidarr-extended#205] [Q&A] [answered]
I'm seeing a "Skipping until all videos are processed for the artist" scroll by in the logs. I only have about a dozen artists tagged in lidarr as "video" and pass that tag to the container as being the only artist to video search for.
I see they have all been processed, yet no music is being downloaded for other artists now.
Just updated the container and watched the log again and seeing the message still.
Any advice or have I configured something incorrectly?

**Comment by RandomNinjaAtk** [comment, marked as answer]
This is normal, is the video script running?  

This might be a bug when using tags, I've never used tags... If you stop using tags, does it work normally?

**Reply by astro7467** [reply]
I see it has downloaded some music for the tagged artists. Will clear the tag flag for videos and restart and report back. 
Video script is running based on logs. 
Thanks.

**Reply by astro7467** [reply]
Tested by disabling the tag check, which meant the video script could start processing artists, so music downloads started after each artist was processed. For now I have disabled video downloads, so all artists are being processed.
When I get a chance will review the code and see where the check is needed so using a tag works so music checks !tag if the flag was set (instead of checking if processed)
BTW ytb-dl has a bug now induced by YT's @ channel name implementation, unsure if the fix has made it to release yet, but basically no YT videos will get downloaded.
